src/components/banner/banner.jsx

```diff
@@ -1,137 +1,158 @@
-import { h, Component } from 'preact';
+import {h, Component} from 'preact';
 import style from './banner.less';
 import Label from '../label/label';
-import Panel from '../panel/panel';
 import ChevronIcon from '../chevronicon/chevronicon';
 
 class LocalLabel extends Label {
-	static defaultProps = {
-		prefix: 'banner'
-	};
+    static defaultProps = {
```

Comment:
Looks like indentation got changed to spaces in this file.  Let's change it back to tabs to stay consistent.